package.json

```diff
@@ -3,44 +3,42 @@
   "version": "2.7.1",
   "private": true,
   "scripts": {
-    "test": "mocha --compilers coffee:coffee-script/register test/*",
+    "test": "mocha --require coffeescript/register test/*",
```

Comment:
Mocha's deprecating the `--compilers` flag, so I went ahead and made the change. 🤷‍♂️